lab_3_keywords_textrank/main.py

```diff
@@ -120,7 +129,9 @@ def _learn_indices(self, tokens: tuple[str, ...]) -> None:
             tokens : tuple[str, ...]
                 sequence of string tokens
         """
-        pass
+        self._word2id = {token: 1000 + index for index, token in enumerate(tokens)}
```

Comment:
looks like you can replace value for a token with a new value. It is not a problem, but better enumerate by set of tokens

Reply:
Done